aquarius/events/events_monitor.py

```diff
@@ -171,7 +176,7 @@ def process_current_blocks(self):
             return
 
         from_block = last_block
```

Comment:
what if from_block=last_block + 1 ?
This way we will never process the same block again  (this is happening now)